src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -127,9 +132,10 @@ public PublicationController(PublicationService publicationService) {
      */
     private void validateRequestHeaders(String provenance, String sourceArtefactId) {
         if (provenance.isEmpty()) {
-            throw new EmptyRequestHeaderException(PublicationConfiguration.PROVENANCE_HEADER);
+            throw new HeaderValidationException(PublicationConfiguration.PROVENANCE_HEADER);
```

Comment:
can keep as EmptyRequestHeaderException for clarity, but now youve created a HeaderValidationException you only need to handle the 1 but can extend for other exceptions that fall under its wing

Reply:
re-added the old exceptions as requested :)